### PR TITLE
add enterprise SSO private preview docs

### PR DESCRIPTION
**Pull request by MichaelTrestman**

### CODE REVIEW

File: cockroachcloud/cloud-sso.md [hunk 1]
```diff
@@ -5,16 +5,62 @@ toc: true
...
+
+### How to enable Enhanced SSO for {{ site.data.products.dedicated }}
+
+File a [support ticket](https://support.cockroachlabs.com/) to enquire about the Enhanced SSO Private Preview.
+
+The following information will be required:
+
+- Customer name
+- CC Organization Id
+- Do you need auto-provisioning? [Yes / No]
```
Comment (kathancox) [inline cockroachcloud/cloud-sso.md:42]: "auto-provisioning" or "Autoprovisioning" Have we set a precedent anywhere else in the docs on the hyphenation here?

File: cockroachcloud/cloud-sso.md [hunk 2]
```diff
@@ -5,16 +5,62 @@ toc: true
...
+This feature allows organization admins to share an organization-specific sign-up/sign-in URL. This removes the need to invite users to your organization.
+
+### How to enable Enhanced SSO for {{ site.data.products.dedicated }}
+
+File a [support ticket](https://support.cockroachlabs.com/) to enquire about the Enhanced SSO Private Preview.
+
+The following information will be required:
+
+- Customer name
+- CC Organization Id
```
Comment (kathancox) [inline cockroachcloud/cloud-sso.md:41]: Nit: "Id" --> "ID"?

File: cockroachcloud/cloud-sso.md [hunk 3]
```diff
@@ -5,16 +5,62 @@ toc: true
...
+- Do you need auto-provisioning? [Yes / No]
+- Specific login methods to enable (any not mentioned here will be disabled)
+- Identity provider to setup [Okta / Active Directory / Azure Active Directory / Onelogin / Other]
+
+- If using SAML, provide **one of** the following options:
```
Comment (kathancox) [inline cockroachcloud/cloud-sso.md:46]: Maybe "Sign-in URL" with the hyphen.

File: cockroachcloud/cloud-sso.md [hunk 4]
```diff
@@ -5,16 +5,62 @@ toc: true
...
+	- (Option 1): The customer IDP metadata XML file
+	- (Option 2): All of the following:
+		- Sign in URL
+		- Signing certificate
+- If using OIDC, provide all of the following:
+	- Issuer URL (ends with .well-known/openid-configuration)
+	- Client ID
+	- Client Secret (If the customer prefers using a frontend channel, then this value won’t be needed)
+		{{site.data.alerts.callout_danger}}
+		This value should be treated as moderately sensitive and so should be passed to the Identity team via OnePassword
```
Comment (kathancox) [inline cockroachcloud/cloud-sso.md:56]: Missing a period at the end of this warning.

File: cockroachcloud/cloud-sso.md [hunk 5]
```diff
@@ -5,16 +5,62 @@ toc: true
...
+		{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_success}}
+Initial response to the ticket will have a 2 business days SLA. After Enhanced SSO  is enabled, the support team will reach out to again to finalize and confirm the setup.
```
Comment (kathancox) [inline cockroachcloud/cloud-sso.md:60]: I think there is a double space between "SSO" and "is enabled" 

Do we need to mention SLA here? (Might be some reason I'm not aware of.) If not, maybe something like: 
"You can expect a response to your ticket [in/within] two business day." 

Also there is an extra "to" between "again" and "finalize" here.

**Review by kathancox — APPROVED** [review]
LGTM with the few typos/nits taken care of. Other items are optional

**Review by MichaelTrestman — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14395)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @abhinavg6, @kathancox, and @MichaelTrestman)

___
*[`cockroachcloud/cloud-sso.md` line 18 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-18:-N5uKqm8EdAmLMnFSr05:boskru9) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L18)):*
<details><summary><i>Previously, kathancox (Kathryn Hancox) wrote…</i></summary><blockquote>

"Enhanced SSO features" seems pretty good to me. The only other option I thought of was "Private Preview SSO features". If these are eventually added to our GA features, would we still refer to them as a set of "enhanced" SSO features, or just kind of part of our offering? 

So, if we went the latter way:

"How to enable Private Preview SSO for {{ site.data.products.dedicated }}"

And: "Additional [SSO features are currently available in Private Preview](#private-preview-enhanced-sso-features)."
</blockquote></details>

Yeah that's pretty good, 'additional' is more neutral than 'enhanced' which sounds more like a breakfast cereal ad campaign!

**Review by MichaelTrestman — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14395)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @abhinavg6, @kathancox, @kpatron-cockroachlabs, and @MichaelTrestman)

___
*[`cockroachcloud/cloud-sso.md` line 15 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-15:-N66H-M45o8utH_jXfOB:bm22xvc) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L15)):*
<details><summary><i>Previously, kathancox (Kathryn Hancox) wrote…</i></summary><blockquote>

I know this was already here, but I think "tie" might need to change to "tying" IIUC. As in: "Administrators avoid .... tying those to other identities and credentials."
</blockquote></details>

I think it's grammatically correct as is. Administrators avoid having to do two things:
- manage...
- tie...
No?
___
*[`cockroachcloud/cloud-sso.md` line 18 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-18:-N66HHtQ07XolsPk_qrt:b-255mzb) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L18)):*
<details><summary><i>Previously, MichaelTrestman (Michael Trestman) wrote…</i></summary><blockquote>

Yeah that's pretty good, 'additional' is more neutral than 'enhanced' which sounds more like a breakfast cereal ad campaign! 
</blockquote></details>

I got the new naming scheme stipulated by product. I dislike it but my objection was overruled. 
___
*[`cockroachcloud/cloud-sso.md` line 28 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5qIbkJBfpSAjfoeOdr:-N69Tl-d6o6zIxRI5qJF:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L28)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

Should probably mention the custom login page above in SAML / OIDC section - added a comment there. And then here, we could mention something like:
"Enterprise authentication allows {{ site.data.products.db }} organization admins to enable or disable specific authentication methods. Once configured, only enabled authentication methods are shown on the sign-in page at the private URL."
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 32 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5qHohw6Jqb6EVu0WEz:-N69U2td2bFk6I9KFvm_:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L32)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

Yeah, we could mention something like:
"With Enterprise Authentication, organization admins can share the private sign-in URL with other users, and optionally configure auto-provisioning. If that's set up, a user's profile is automatically created in the CockroachDB Cloud organization when they sign-in for the first time using an enabled SSO authentication method."
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 42 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-42:-N69g5xo-_c4d9Kybws7:b-ho6e3i) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L42)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

> See comment about removing the "requested information"
See comment about removing the "requested information"
</blockquote></details>

Neither word appears elsewhere in the docs (surprisingly?). Let's go with the hyphen, rather than making up the new word.
___
*[`cockroachcloud/cloud-sso.md` line 46 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-46:-N69gSiaARxciVrJTapM:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L46)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

See comment about removing the "requested information"
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 56 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-56:-N69gZTXAQE-Oz0nGY_T:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L56)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

See comment about removing the "requested information"
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 56 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5qIBTq7HvMHfr2mMRk:-N69gazB6qNhRRcxMvUG:b-kzu56u) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L56)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

See comment about removing the "requested information"
</blockquote></details>

@abhinavg6 @kpatron-cockroachlabs
I have no way of knowing what is for CRL eyes only. I'm going off that one source doc, so please do make sure there is nothing else that should be chopped or is inaccurate.

In the future, I think it would be better if docs team could be provided with specs that are accurate for what should be put in the docs. This ticket has been a lot of back and forth with me trying to imaginatively infer the desired content of the docs from the internal doc not prepared for the purpose, and iteratively clarifying through negative feedback. This is a very inefficient process
___
*[`cockroachcloud/cloud-sso.md` line 60 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-60:-N69hZ0BBlgP29sWRN1I:b-hajau7) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L60)):*
<details><summary><i>Previously, kpatron-cockroachlabs (Kyle) wrote…</i></summary><blockquote>

I would also suggest dropping the SLA language. I believe that was copied from an internal doc.
</blockquote></details>

Yes. Literally everything came from that internal doc and I have no idea what is supposed to go in and what not unless you tell me. @abhinavg6   

Dropped the SLA language, but should we still set the expectation that 2 days is the expected response time? Keeping it unless I hear.
___
*[`cockroachcloud/cloud-sso.md` line 22 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67j_Mh7LkgRRU36LZa:-N69LWoN9W5tQ1VjvKth:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L22)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

"..allow internal applications and external service providers to rely on standard IdPs for centralized authentication".
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 34 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67puc17xGXagiBSGTn:-N69U58lDbHfKSfT6PIO:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L34)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

Shouldn't "Enhanced SSO" be "Enterprise Authentication"?
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 36 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67qEmD6xPAumLFwf60:-N69UBi120ToHlhjoeQ_:b2p1xpz) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L36)):*
> "Reach out to your Cockroach Labs team to enquire about the private preview of Enterprise Authentication for Cloud SSO
Don't you only have a "Cockroach Labs team" if you are a dedicated customer? This is supposedly available for serverless customers.
If I'm a serverless customer, how do I "reach out to the Cockroach Labs team" other than by filing a support ticket? Do we have a link or something? 

I could link to the "contact us page" which just has this email address: info@cockroachlabs.com, or provide that. 
___
*[`cockroachcloud/cloud-sso.md` line 38 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67qVcHEn5cF0jW1Ty3:-N69fDey1qcEfj0ridaK:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L38)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

I think we should remove all the requested information for now, and leave that for internal wiki for account teams. We can come back to this once we've the self-service UI for the this feature. @kpatron-cockroachlabs  what do you think?
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 60 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67qy3qByxaRCGR8i-r:-N69hs-uBNKizgMyWoSu:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L60)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

See comment about removing the "requested information"
</blockquote></details>

Done.
___
*[`v22.1/sso.md` line 2 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67r4mU-gwwlQBkzQ5H:-N69iJzM4sh6kKb72d-E:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/v22.1/sso.md#L2)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

"access" is implied with SSO - could remove that word
</blockquote></details>

Done.
___
*[`v22.1/sso.md` line 3 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67r8fTExN7qZfwjTjk:-N69iKki1TJ6g-llHK57:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/v22.1/sso.md#L3)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

remove "access"
</blockquote></details>

Done.
___
*[`v22.1/sso.md` line 13 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67rG2A4zKSJ0ipmSgV:-N69ijQ--kfHzBj7GYUS:bth74vr) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/v22.1/sso.md#L13)):*
> { site.data.products.db }}
"{ site.data.products.db }}" renders to CockroachDB Cloud.

**Review by MichaelTrestman — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14395)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @abhinavg6, @kathancox, and @kpatron-cockroachlabs)

___
*[`cockroachcloud/cloud-sso.md` line 8 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-8:-N69s_YH3-1rhE6mYLHC:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L8)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

+1
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 3 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67hUECC-ZLltk48LzL:-N69sUMXDXyDTyUTljnT:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L3)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

 "Cockroach Cloud Console" -> "CockroachDB Cloud Console"
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 10 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67iBRFAWG6uRUJiIQf:-N69sQcz8_dRYwGwZcah:bbwiu2x) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L10)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

I would suggest that we use "Reach out to your Cockroach Labs team to enquire about the private preview of Enterprise Authentication for Cloud SSO." instead of "File a [support ticket](https://support.cockroachlabs.com/) to enquire about the Enterprise Authentication for Cloud SSO Private Preview."
</blockquote></details>

"Reach out to your Cockroach Labs team to enquire about the private preview of Enterprise Authentication for Cloud SSO

Don't you only have a "Cockroach Labs team" if you are a dedicated customer? This is supposedly available for serverless customers.
If I'm a serverless customer, how do I "reach out to the Cockroach Labs team" other than by filing a support ticket? Do we have a link or something?

I could link to the "contact us page" which just has this email address: info@cockroachlabs.com, or provide that.

**Review by abhinavg6 — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14395)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kathancox, @kpatron-cockroachlabs, and @MichaelTrestman)

___
*[`cockroachcloud/cloud-sso.md` line 56 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5qIBTq7HvMHfr2mMRk:-N6BXuVd9S_oY2ejaUH8:br65qmd) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L56)):*
<details><summary><i>Previously, MichaelTrestman (Michael Trestman) wrote…</i></summary><blockquote>

@abhinavg6 @kpatron-cockroachlabs
I have no way of knowing what is for CRL eyes only. I'm going off that one source doc, so please do make sure there is nothing else that should be chopped or is inaccurate.

In the future, I think it would be better if docs team could be provided with specs that are accurate for what should be put in the docs. This ticket has been a lot of back and forth with me trying to imaginatively infer the desired content of the docs from the internal doc not prepared for the purpose, and iteratively clarifying through negative feedback. This is a very inefficient process
</blockquote></details>

Thanks for the feedback Michael, and my apologies for the frustration with this one. We'll make sure that it's a more seamless process in future.
___
*[`cockroachcloud/cloud-sso.md` line 60 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-60:-N6BWY1N3MPc1vriep0Y:b-2ec4ch) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L60)):*
<details><summary><i>Previously, MichaelTrestman (Michael Trestman) wrote…</i></summary><blockquote>

Yes. Literally everything came from that internal doc and I have no idea what is supposed to go in and what not unless you tell me. @abhinavg6   

Dropped the SLA language, but should we still set the expectation that 2 days is the expected response time? Keeping it unless I hear.
</blockquote></details>

Thanks Michael. We should remove the 2 days language from here - that's for internal purposes. I've added a suggestion comment to this.
___
*[`cockroachcloud/cloud-sso.md` line 10 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67iBRFAWG6uRUJiIQf:-N6BTQ4a9-qV49IQ7zbV:b78oghf) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L10)):*
<details><summary><i>Previously, MichaelTrestman (Michael Trestman) wrote…</i></summary><blockquote>

"Reach out to your Cockroach Labs team to enquire about the private preview of Enterprise Authentication for Cloud SSO

Don't you only have a "Cockroach Labs team" if you are a dedicated customer? This is supposedly available for serverless customers.
If I'm a serverless customer, how do I "reach out to the Cockroach Labs team" other than by filing a support ticket? Do we have a link or something?

I could link to the "contact us page" which just has this email address: info@cockroachlabs.com, or provide that.
</blockquote></details>

Even Serverless customers who would want to integrate with an enterprise IdP would've a CRL account team. But yeah the ones which are just testing it on their own may not. Let's then combine both statements - one including the reach out to CRL team as well as the one for filing a support ticket, but in a way where we recommend the former and latter is a fallback.
___
*[`cockroachcloud/cloud-sso.md` line 36 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67qEmD6xPAumLFwf60:-N6BV6CDDpSRrUkJFgYD:befvba5) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L36)):*
<details><summary><i>Previously, MichaelTrestman (Michael Trestman) wrote…</i></summary><blockquote>

> "Reach out to your Cockroach Labs team to enquire about the private preview of Enterprise Authentication for Cloud SSO
Don't you only have a "Cockroach Labs team" if you are a dedicated customer? This is supposedly available for serverless customers.
If I'm a serverless customer, how do I "reach out to the Cockroach Labs team" other than by filing a support ticket? Do we have a link or something? 

I could link to the "contact us page" which just has this email address: info@cockroachlabs.com, or provide that. 
</blockquote></details>

Same response as above. Let's combine the statements to provide both CRL team and support as mechanisms.
___
*[`cockroachcloud/cloud-sso.md` line 38 at r6](https://reviewable.io/reviews/cockroachdb/docs/14395#-N6BXlap1b1xOvsb54N9:-N6BXlap1b1xOvsb54NA:bgs6dtr) ([raw file](https://github.com/cockroachdb/docs/blob/08141b294cd559cd5100a708b4c8ee0f74c610e7/cockroachcloud/cloud-sso.md#L38)):*
> ```Markdown
> ### Auto-provisioning
> 
> This optional enterprise feature removes the need to invite users to your organization. When auto-provisioning is enabled, a {{ site.data.products.db }} account will be created automatically for your users when they sign in to the [custom sign-in page](#custom-sign-in-page) for the first time.
> ```

Would it be possible to link "invite users to your organization" to https://www.cockroachlabs.com/docs/cockroachcloud/console-access-management.html#invite-team-members-to-cockroachdb-cloud?
___
*[`cockroachcloud/cloud-sso.md` line 45 at r6](https://reviewable.io/reviews/cockroachdb/docs/14395#-N6BVK5h34vErerD3I_2:-N6BVK5h34vErerD3I_3:bvtqu9g) ([raw file](https://github.com/cockroachdb/docs/blob/08141b294cd559cd5100a708b4c8ee0f74c610e7/cockroachcloud/cloud-sso.md#L45)):*
> ```Markdown
> 
> {{site.data.alerts.callout_success}}
> You should hear back from support within 2 business days.
> ```

Once you make the change to combine the contact options above, maybe slightly tweak it to indicate:
"You should hear back from your Cockroach labs team or from support soon."

I would hesitate to share the internal SLA in public doc, as that would make it an external SLA.
___
*[`cockroachcloud/cloud-sso.md` line 47 at r6](https://reviewable.io/reviews/cockroachdb/docs/14395#-N6BWAFL4iwZaH0J7TRH:-N6BWAFL4iwZaH0J7TRI:bw0n8lp) ([raw file](https://github.com/cockroachdb/docs/blob/08141b294cd559cd5100a708b4c8ee0f74c610e7/cockroachcloud/cloud-sso.md#L47)):*
> ```Markdown
> You should hear back from support within 2 business days.
> 
> After Enhanced SSO is enabled, the support team will reach out to again to finalize and confirm the setup.
> ```

"After Enterprise Authentication is enabled, your Cockroach Labs team or the support team will reach out again to finalize and confirm the setup."

**Review by MichaelTrestman — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14395)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @abhinavg6, @kathancox, and @kpatron-cockroachlabs)

___
*[`cockroachcloud/cloud-sso.md` line 56 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5qIBTq7HvMHfr2mMRk:-N6FNaLq7baWq5a3nRG6:b-54avio) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L56)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

Thanks for the feedback Michael, and my apologies for the frustration with this one. We'll make sure that it's a more seamless process in future.
</blockquote></details>

Ok, thanks.
___
*[`cockroachcloud/cloud-sso.md` line 60 at r3](https://reviewable.io/reviews/cockroachdb/docs/14395#-N5p-fqhaMN-df6lFJ59-r3-60:-N6FNvToAEZpy-dRbvym:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2e1fcf5ae38fd26ab65b6f925bc69d96294df565/cockroachcloud/cloud-sso.md#L60)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

Thanks Michael. We should remove the 2 days language from here - that's for internal purposes. I've added a suggestion comment to this.
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 10 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67iBRFAWG6uRUJiIQf:-N6FOLpi8kZa6Id_V7TT:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L10)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

Even Serverless customers who would want to integrate with an enterprise IdP would've a CRL account team. But yeah the ones which are just testing it on their own may not. Let's then combine both statements - one including the reach out to CRL team as well as the one for filing a support ticket, but in a way where we recommend the former and latter is a fallback.
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 36 at r4](https://reviewable.io/reviews/cockroachdb/docs/14395#-N67qEmD6xPAumLFwf60:-N6FNZhPBEq2ZmLfL_c8:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/2c5f7da26c4329490e8dbace0dbb5e507b13c468/cockroachcloud/cloud-sso.md#L36)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

Same response as above. Let's combine the statements to provide both CRL team and support as mechanisms.
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 38 at r6](https://reviewable.io/reviews/cockroachdb/docs/14395#-N6BXlap1b1xOvsb54N9:-N6FNVdt00fKwdv4la-v:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/08141b294cd559cd5100a708b4c8ee0f74c610e7/cockroachcloud/cloud-sso.md#L38)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

Would it be possible to link "invite users to your organization" to https://www.cockroachlabs.com/docs/cockroachcloud/console-access-management.html#invite-team-members-to-cockroachdb-cloud?
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 45 at r6](https://reviewable.io/reviews/cockroachdb/docs/14395#-N6BVK5h34vErerD3I_2:-N6FNsEfELv-1zDAiyaP:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/08141b294cd559cd5100a708b4c8ee0f74c610e7/cockroachcloud/cloud-sso.md#L45)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

Once you make the change to combine the contact options above, maybe slightly tweak it to indicate:
"You should hear back from your Cockroach labs team or from support soon."

I would hesitate to share the internal SLA in public doc, as that would make it an external SLA.
</blockquote></details>

Done.
___
*[`cockroachcloud/cloud-sso.md` line 47 at r6](https://reviewable.io/reviews/cockroachdb/docs/14395#-N6BWAFL4iwZaH0J7TRH:-N6FOINT6WDOmb6YpzC-:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/08141b294cd559cd5100a708b4c8ee0f74c610e7/cockroachcloud/cloud-sso.md#L47)):*
<details><summary><i>Previously, abhinavg6 (Abhinav Garg) wrote…</i></summary><blockquote>

"After Enterprise Authentication is enabled, your Cockroach Labs team or the support team will reach out again to finalize and confirm the setup."
</blockquote></details>

Done.

**Review by MichaelTrestman — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14395#-:-N6FOfPj6OqrJ5xCyHMP:bnfp4nl)* status: :shipit: complete! 0 of 0 LGTMs obtained (and 1 stale) (waiting on @abhinavg6, @kathancox, and @kpatron-cockroachlabs)

**Pull request merged**